ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -39,6 +39,15 @@ VAV_SD_DSP:
   - SD
   - DSP
 
+VAV_SD_ZTC_CSPO:
+  description: "Simple cooling only VAV with occupancy."
+  is_canonical: true
+  implements:
+  - VAV
+  - SD
+  - ZTC
```

Comment:
ZTC includes `zone_air_temperature_setpoint`, and your other type defines `zone_air_occupied_cooling_temperature_setpoint` and `zone_air_unoccupied_cooling_temperature_setpoint`.  We shouldnt have all of these setpoints. If you end up with an effective zone temp setpoint, like `zone_air_cooling_temperature_setpoint` then that would be the only one we need. You could then change `CSP` to include `occupancy_status` as an optional field and use that.

Reply:
but if you want both setpoints, then we would have [this type](https://github.com/google/digitalbuildings/blob/accdc3b2685b08531cd83fcdd048f432bb2a2008/ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml#L639) you could reference. Update it for cooling and add occupancy status as required (and keep the naming convention intact).

Reply:
We do have an effective set point and hence have modeled as per your suggestion and added occupancy status as an optional field in CSP.